pkg/jsx/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@hyperapp/jsx",
+  "description": "Hyperapp JSX runtime",
```

Comment:
Let's go with something like: "JSX meets Hyperapp" for now. 😄

Reply:
Updated